README.md

```diff
@@ -25,17 +25,29 @@ Unified API for music players.
 #### Linux
 
 - [x] [MPRIS](https://specifications.freedesktop.org/mpris-spec/latest/) (test needed) (Thanks to [@suransea](https://github.com/suransea))
+- [x] MPRIS Now Playing: Just like Now Playing, but automatically find available MPRIS players.
 
 ##### dependencies
 
 - [playerctl](https://github.com/altdesktop/playerctl) (could be installed by package manager)
 
+> A running `GMainLoop` is required to automatically update the player and playback status for MPRIS.
+> If not, you can run one by: 
+> ```swift
+> GRunLoop.main.run()
+> ```
+> or in other threads:
+> ```swift
+> Thread.detachNewThread { 
+>     GRunLoop()!.run() 
```

Comment:
I didn't use Glib much. It seems you're running a thread run loop. According to documentation:

> g_main_context_get_thread_default()
> ... Asynchronous operations that want to be able to be run in contexts other than the default one should call this method or g_main_context_ref_thread_default() to get a GMainContext to add their GSources to.

Would playerctl able to use this run loop?

Reply:
Playerctl use this runloop while the thread has no default context, which the `g_main_context_ref_thread_default` equals to `g_main_context_default`. I should change the example to use `GRunLoop.main.run()` since the initializer useful for other cases.